docs/modules/ups_userguide/pages/server-installation.adoc

```diff
@@ -37,6 +38,29 @@ export APPSRV_HOME=/path/to/SERVER_HOME
 set APPSRV_HOME=X:\path\to\SERVER_HOME
 ----
 
+[[quickstart]]
+=== Quickstart Guide
+
+Build the project:
+[source,shell]
+----
+mvn clean install
+----
+
+This builds the latest UPS image which can be run with Keycloack or noauth. 
```

Comment:
Maybe: `This builds and installs the push server to your local maven registry and also builds two docker images, plain (no authentication) and kc (includes Keycloak)`